iapetus/iapetus.py

```diff
@@ -255,9 +269,9 @@ def _create_thermodynamic_states(self, reference_thermodynamic_state, spacing=0.
         K_xz = self.kT / (sigma_xz**2) # spring constant
         print('in-plane sigma_xz = {:.3f} A'.format(sigma_xz / unit.angstroms))
 
-        dr = axis_distance * (expansion_factor - 1.0)/2.0
+        dr = axis_distance * (expansion_factor - 1.0)/1.0 # Modified by Ana
         rmax = axis_distance + dr
-        rmin = - dr
+        rmin = -1.0 * axis_distance # Modified by Ana
```

Comment:
No need to leave a `# Modified by Ana` comment since we can see your changes in the commit diffs!